Altis_Life.Altis/core/shops/fn_chopShopMenu.sqf

```diff
@@ -2,25 +2,33 @@
 /*
     File: fn_chopShopMenu.sqf
     Author: Bryan "Tonic" Boardwine
+    Modified: Casperento
```

Comment:
See the comment on the conversation here: <https://github.com/AsYetUntitled/Framework/pull/564/files/a33681c53b384b2aaa97286df87bbae6fc5f2491#diff-480bd50ccc3ed591fc3ed0bd4b402585>

I've made the same point a few times in recent PRs

Reply:
No problem, i'll remove these 'Modified' comments